api/db/repositories/orderHelpers/orderHelpers.js

```diff
@@ -0,0 +1,99 @@
+const orderDomain = require('@domains/ordersDomain')
+const orderRepo = require('@repos/orderRepository')
+const productRepo = require('@repos/productRepository')
+const { Verification, Resources } = require('@verify/verification')
+
+class OrderHelpers {
+    static async AddCounters(order, productId, quantity) {
+        const product = await productRepo.getById(productId)
+        
+        const orderDataValues = order.dataValues
+        let productsCount = orderDataValues.productsCount
+        let productsTotalPrice = orderDataValues.productsTotalPrice
+
+        switch (order.currency) {
+            case 'pln':
```

Comment:
actually instead of using switch case here u can use some JS goodies. just do
``` js
productsTotalPrice += product.price[order.currency] * quantity
```